source/LoadPanel.cpp

```diff
@@ -71,17 +71,17 @@ namespace {
 	string TimestampString(time_t timestamp)
 	{
 		pair<const char*, const char*> format = TimestampFormatString(Preferences::GetDateFormat());
-		stringstream ss;
+		static const size_t BUF_SIZE = 26;
+		char str[BUF_SIZE + 1];
```

Comment:
~~actual last thing~~
Using the `char *, size_t` constructor of `std::string` means we don't need to null terminate the character array.
By my count, the longest the timestamp string could possibly be is 24 characters:
`yyyy-mm-dd hh:mm:ss` always 19
`hh:mm pm on dd MMM yyyy` 21-23
`hh:mm pm on MMM dd, yyyy` 22-24
```suggestion
		static const size_t BUF_SIZE = 24;
		char str[BUF_SIZE];
```